src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java

```diff
@@ -48,37 +49,73 @@ public CallbackProcessor(CoreCaseDataApi ccdApi, CcdAuthenticatorFactory authFac
 
     private List<String> attachCase(String theType,
                                     String anEventId,
-                                    String jurisdiction,
+                                    String exceptionRecordJurisdiction,
                                     String caseTypeId,
                                     String caseRef,
                                     CaseDetails exceptionRecord) {
-        CcdAuthenticator authenticator = authFactory.createForJurisdiction(jurisdiction);
         try {
-            CaseDetails theCase = retrieveCase(caseRef, authenticator);
+            return attachCase(exceptionRecordJurisdiction, caseRef);
+        } catch (CallbackException e) {
+            String message = e.getMessage();
+            log.error(message, e);
+            return ImmutableList.of(message);
+        }
+    }
+
+    @NotNull
+    private List<String> attachCase(String exceptionRecordJurisdiction, String caseRef) {
+        CcdAuthenticator authenticator = authFactory.createForJurisdiction(exceptionRecordJurisdiction);
+        CaseDetails theCase = getCase(caseRef, authenticator);
+        StartEventResponse event = startAttachScannedDocs(caseRef, authenticator, theCase);
+        return emptyList();
+    }
+
+    private StartEventResponse startAttachScannedDocs(String caseRef, CcdAuthenticator authenticator, CaseDetails theCase) {
+        try {
+            return startAttachScannedDocs(caseRef, authenticator, theCase.getJurisdiction(), theCase.getCaseTypeId());
+        } catch (FeignException e) {
+            throw error(e, "Internal Error: start event call failed case: %s Error: %s", caseRef, e.status());
+        }
+    }
+
+    private CaseDetails getCase(String caseRef, CcdAuthenticator authenticator) {
+        try {
+            return retrieveCase(caseRef, authenticator);
         } catch (FeignException e) {
             switch (e.status()) {
                 case 404:
-                    return error("Could not find case: %s", caseRef);
+                    throw error(e, "Could not find case: %s",
+                        caseRef);
                 default:
-                    return error("Internal Error: Could not retrieve case: %s Error: %s", caseRef, e.status());
+                    throw error(e, "Internal Error: Could not retrieve case: %s Error: %s",
+                        caseRef, e.status());
```

Comment:
and here. if it's too long, perhaps each parameter in it's own line?

Reply:
I don't really like that style since it takes a lot of space ... but can do if its required